bigbluebutton-html5/imports/api/2.0/cursor/server/handlers/cursorUpdate.js

```diff
@@ -0,0 +1,15 @@
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+import updateCursor from '../modifiers/updateCursor';
+
+export default function handleCursorUpdate({ header, body }) {  
+  const meetingId = header.meetingId;
```

Comment:
Cursors also need the userId from the header to identify it. And when the cursor gets displayed we show their name to identify the cursor. I don't know if you want to look up the name and store it once or look it up every time, but that's the usage.